applications/advanced-concepts/tags-and-placeholders.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Tags and placeholders in machine translation
+---
+
+# Tags and placeholders
+
+In-line **tags and placeholders** can negatively impact the machine translation quality when the engine cannot interpret what they are. In many cases, the machine translation treats these marks as delimiters or ordinary words, misplacing, corrupting or ignoring them.
```

Comment:
First say what they are.

Then talk about the problems with them.

I think the problems can be divided into roughly two types:
- the tags getting corrupted, misplaced or ignored
- the translation quality (regardless of the tags)

Reply:
No sentence should be this long.

Reply:
Could you please let me know what you meant with "the translation quality (regardless of the tags)"? I think the problem is that, when tags and placeholders get converted into their end content, the machine translation has already been applied and forced to make certain morphological decisions.

Reply:
There are 2 approaches to preserving tags.
1. leave them in, santiguarse, and let the MT learn to handle it
2. remove, then reinsert (at both training and inference time)

You're assuming 2.  But 1 is back in fashion, and it can degrade the output.  Concretely, today, 1 is used by the top systems and if you add tags around every word, the translation quality is terrible.